internal/querycoordv2/meta/replica_manager.go

```diff
@@ -261,16 +261,18 @@ func (m *ReplicaManager) GetByNode(nodeID typeutil.UniqueID) []*Replica {
 }
 
 func (m *ReplicaManager) getByCollectionAndRG(collectionID int64, rgName string) []*Replica {
-	m.rwmutex.RLock()
```

Comment:
getByCollectionAndRG is a private method now, will always be protected by lock at caller.